src/emqx_ct_helpers_docker.erl

```diff
@@ -16,7 +16,7 @@
 
 -module(emqx_ct_helpers_docker).
 
--export([docker_ip/2, compose/5, stop/1]).
+-export([docker_ip/2, compose/5, stop/1, remove/1]).
```

Comment:
sorry I still don't have the full picture
is the usage always like stop then remove? if yes, we do not need the stop API,
a `-f` option for `docker rm` will force it.
i.e. only need `remove/1` ?

Reply:
yes, the current usage of in the only Suite using it is stopping & then removing. The module as titled "...helpers_docker" is meant to provide some reasonable re-usable functions for other who will be using the module in more nuanced ways from other Suites.
Adding an additional force_remove function would be good for my current usage.

Reply:
Added